twarc.py

```diff
@@ -466,8 +466,8 @@ def timeline(self, user_id=None, screen_name=None, max_id=None,
             try:
                 resp = self.get(url, params=params, allow_404=True)
             except requests.exceptions.HTTPError as e:
-                if e.response.status_code == 404:
-                    logging.info("no timeline available for %s", id)
+                if e.response.status_code in [401, 404, 502, 503, 504]:
```

Comment:
Hi Todd, have you actually seen 502, 503 and 504 errors coming back from the API?